app/src/main/java/hous/release/android/presentation/profile/edit/ProfileEditActivity.kt

```diff
@@ -0,0 +1,139 @@
+                finish()
+            } else {
+                // 에러 시에 띄울 뷰
+            }
+        }
+    }
+
+    private fun initSaveBtnColor() {
+        profileEditViewModel.nickname.observe(this) { nickname ->
+            if (nickname.isNullOrEmpty()) {
+                binding.btnProfileEditSave.setTextColor(getColor(R.color.hous_g_4))
+            } else {
+                binding.btnProfileEditSave.setTextColor(getColor(R.color.hous_blue))
+            }
+        }
+    }
+
+    private fun initBackBtnOnClickListener() {
+        binding.btnProfileEditBack.setOnClickListener {
+            WarningDialogFragment().apply {
+                arguments = Bundle().apply {
+                    putSerializable(
+                        WarningDialogFragment.WARNING_TYPE,
+                        WarningType.WARNING_EDIT_PROFILE
+                    )
+                    putParcelable(
+                        WarningDialogFragment.CONFIRM_ACTION,
+                        ConfirmClickListener(confirmAction = { finish() })
+                    )
+                }
+            }.show(supportFragmentManager, WarningDialogFragment.DIALOG_WARNING)
+        }
+    }
+
+    private fun initBirthdayOnClickListener() {
+        binding.etProfileEditBirthday.setOnClickListener {
+            val birthdayInt = initDateFormatToInt(profileEditViewModel.birthday.value!!)
+            DatePickerDialog().apply {
+                arguments = Bundle().apply {
+                    putParcelable(
+                        WarningDialogFragment.CONFIRM_ACTION,
+                        DatePickerClickListener(
+                            confirmActionWithDate = { date -> initDate(date) },
+                            birtyYear = birthdayInt[0],
+                            birthMonth = birthdayInt[1],
+                            birthDay = birthdayInt[2]
+                        )
+                    )
+                }
+                Timber.e("$birthdayInt")
+            }.show(supportFragmentManager, SELECT_BIRTHDAY)
+        }
+    }
+
+    private fun initBirthdayPublicOnClickListener() {
+        binding.llProfileEditBirthdayPublic.setOnClickListener {
+            with(binding) {
+                tvProfileEditBirthdayPublic.isSelected =
+                    !tvProfileEditBirthdayPublic.isSelected
+                ivProfileEditBirthdayCheck.isSelected =
+                    !ivProfileEditBirthdayCheck.isSelected
+            }
+            profileEditViewModel.initBirthdayPublic(binding.tvProfileEditBirthdayPublic.isSelected)
+        }
+    }
+
+    private fun initDate(date: String) {
+        profileEditViewModel.initSelectedBirthDate(date)
+    }
+
+    private fun initDateFormatToInt(birthday: String): List<Int> {
+        val birthdayList: List<String> = birthday.split(SLASH)
+        val year = birthdayList[0].toInt()
+        val month = birthdayList[1].toInt()
+        val day = birthdayList[2].toInt()
+        Timber.e("$year, $month, $day")
```

Comment:
```suggestion
    private fun initDateFormatToInt(birthday: String) = birthday.split("/").map { it.toInt() }
```  
위와 같이 코드를 줄일 수도 있겠습니다.

Reply:
이렇게 줄이니까 터지네요..